pkgs/by-name/en/ente-auth/package.nix

```diff
@@ -51,8 +52,13 @@ flutter324.buildFlutterApplication rec {
     webkitgtk_4_0
     sqlite
     libayatana-appindicator
+    jdk17_headless # JDK version used by upstream CI
   ];
 
+  preFixup = ''
+    rm $out/app/ente-auth/lib/libdartjni.so
+  '';
```

Comment:
Maybe add a comment explaining how this file got there and not needed?

Nit: this should be after `postInstall`